armotypes/vulnerabilitytypes.go

```diff
@@ -9,11 +21,175 @@ type VulnerabilityJobParams struct {
 	JobID           string `json:"jobID,omitempty"`
 }
 
+type VulnerabilityWorkload struct {
+	Id               string                     `json:"id"`
+	Name             string                     `json:"name"`
+	Namespace        string                     `json:"namespace"`
+	Type             string                     `json:"type"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	ImagesCount      int                        `json:"imagesCount"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	RiskFactors      []RiskFactor               `json:"riskFactors"`
+	Labels           []string                   `json:"labels"`
+}
+
+type SeverityDetails struct {
+	Total   int             `json:"total"`
+	TopCves []TopCveDetails `json:"topCves"`
+}
+
+type TopCveDetails struct {
```

Comment:
Why Top?

Reply:
fetching top is the same as fetching the whole cves list? the goal is to avoid fetching redundant data

Reply:
in my query i bring all cves by severity

Reply:
We fetch all cves name sorted by severity according to the filter (if only critical is on so will be the list of CVEs)